garrysmod/lua/vgui/dcolormixer.lua

```diff
@@ -211,8 +208,7 @@ end
 
 function PANEL:SetColor( color )
 
-	local h, s, v = ColorToHSV( color )
-	self.RGB.LastY = ( 1 - h / 360 ) * self.RGB:GetTall()
+	self.RGB.LastY = ( 1 - ColorToHSV( color ) / 360 ) * self.RGB:GetTall()

```

Comment:
This again is more confusing for new people if we gonna take that angle with PRs, even if it saves 0.0000001ms.

Reply:
I don't see how it's more confusing, and as you said, "who even reads base code anyway"

Reply:
16ea8b772b0b8a488529f65b3a3c947d93a2218a is perhaps easier to understand.